core/global/input_manager.gd

```diff
@@ -185,6 +195,7 @@ func _on_gamepad_change(_device: int, _connected: bool) -> void:
 	if handheld_gamepad and not handheld_gamepad.is_open():
 		if handheld_gamepad.open() != OK:
 			logger.error("Unable to open handheld keyboard device.")
+		hide_event_device(handheld_gamepad.kb_event_path)
```

Comment:
So are we only hiding handheld gamepads? Should we also be hiding regular gamepads..?

Reply:
Regular gamepads get hidden a few lines above. IDK why it's not showing in the diff

https://github.com/ShadowBlip/OpenGamepadUI/blob/60e1733230f57ca499de256cc72dbbf6926f6614/core/global/input_manager.gd#L193